hack/godep-save.sh

```diff
@@ -88,11 +88,11 @@ kube::log::status "Running godep save - this might take a while"
 # This uses $(pwd) rather than ${KUBE_ROOT} because KUBE_ROOT will be
 # realpath'ed, and godep barfs ("... is not using a known version control
 # system") on our staging dirs.
-GOPATH="${GOPATH}:$(pwd)/staging" ${KUBE_GODEP:?} save -i $(IFS=,; echo "${IGNORED_PACKAGES[*]}") "${REQUIRED_BINS[@]}"
+GOPATH="${GOPATH}:$(pwd)/staging" "${KUBE_GODEP:?}" save -i "$(IFS=,; echo "${IGNORED_PACKAGES[*]}")" "${REQUIRED_BINS[@]}"
 
 # create a symlink in vendor directory pointing to the staging client. This
 # let other packages use the staging client as if it were vendored.
-for repo in $(ls staging/src/k8s.io); do
+for repo in "staging/src/k8s.io"; do
```

Comment:
Why modify and what does it fix: 
```
In ./hack/godep-save.sh line 95:
for repo in $(ls staging/src/k8s.io); do
            ^----------------------^ SC2045: Iterating over ls output is fragile. Use globs.
```
Ref: https://www.shellcheck.net/wiki/SC2045 -- Iterating over ls output is fragi...

Reply:
Please test these throughly. This is changing functionality, and breaking the script. You're just passing a single string, rather than a shell glob.

This should actually look like:
```sh
for repo in staging/src/k8s.io/*; do
  repo="${repo#staging/src/k8s.io/}"
  if [ ! -e "vendor/k8s.io/${repo}" ]; then
    ln -s "../../staging/src/k8s.io/${repo}" "vendor/k8s.io/${repo}"
  fi
done
```

Reply:
The modified. thanks.